### ISSUE TITLE
Shelter doesn't work on Samsung Galaxy A03. Uninstalled Shelter. No profile work can be found on Settings. Restarted the phone. Black Screen

### ISSUE THREAD

**Issue by mlocverm** [PeterCxy/Shelter#307]
This is my first time using Shelter, it's my fault, there are lot of things going on here.

So, I'm using Samsung Galaxy A03, trying to see if work profile works (I used it before on my previous samsung phone, with built in apps). I installed Shelter, created the Shelter profile, no notification, not on DnD mode, stuck on waiting to create profile shelter. I waited for hours and still like that. So I tried to uninstall it by following the steps on this repo, but I found no work profile on Settings > Account (even searching on the search bar and there is no "work" for work profile). I cleared Shelter data and uninstalled it.

When I tried to install Shelter from apk file again, there is an option to install it on work profile but it said "blocked by your IT Admins" or something like that. So I restarted my phone, hoping that it would just restart the system and the invisible/blocked work profile would be gone. But I got black screen. How to fix this black screen? should I factory reset my phone? flash my phone? or there are another way?

**Comment by mlocverm** [comment]
I entered recovery mode, Repair App, restart my phone again, and I see that my phone somewhat working again (the black screen is appear but disappear when I turn off and on my screen), but I restarted it again because I panicked, and **it now works fine again, no black screen.**

But the Invisible/blocked profile is still there:
![WhatsApp Image 2024-07-28 at 09 12 15_c590469a](https://github.com/user-attachments/assets/9d72c931-20c7-48aa-a903-31e1060bf6cb)
it's always there on the "Open With" dialog

**Comment by mlocverm** [comment]
I fixed the problem, the black screen was coming from another different app, called FakeStandby. In short, I locked myself out because there is a broken feature on that app called "use 4 taps or more to unlock" and "start on boot". But luckily there is another feature that helps me unlocked the screen.

I was using old Shelter version, and installed work profile. I was unable to update shelter because there is old shelter exist on work profile. So, I removed the broken work profile using guide from this site:
[Bayton.org - Enrolment Failed WP](https://bayton.org/android/android-enterprise-faq/enrolment-failed-delete-wp/)

After I remove the old work profile, I installed the latest shelter version from F-Droid, and created new work profile, and now it's all working fine again. Thank you for making this app. Because knox secure folder is not included on the One UI Core version (or so, they said on reddit)

that's all from me, I will close this issue
